vllm/model_executor/layers/__init__.py

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+from vllm.model_executor.layers.determinism import init_determinism
+
+init_determinism()
```

Comment:
Why is this change done at global init phase? Can we do it somewhere in the model code instead?

Reply:
ok, I moved the init to gpu runner (as Horace had it)